Reference/Querying/MemberShipHelper/index.md

```diff
@@ -1,8 +1,15 @@
 ---
 versionFrom: 8.0.0
+versionTo: 8.0.0
```

Comment:
I am just wondering if we add this then the version box would show as 8.0.0 - 8.0.0 (current) which will make it question if the article is not valid for 8+versions.

Maybe add versionTo: 8.18.4?

Reply:
That's also an option.
Then perhaps let's do 8.18.0 then?
That way, if we release patches then we won't have to go in an update this one every time? 🙈

Reply:
Yup, sounds good 💪